src/command/list/toolchain.rs

```diff
@@ -55,6 +55,17 @@ impl Lookup {
         default_platform: Option<&PlatformSpec>,
         version: &Version,
     ) -> Source {
+        let default_or_none = default_platform
+            .clone()
```

Comment:
suggestion: It looks like this clone is redundant. It was there before, so if we're refactoring it makes sense to also clean that up.

```suggestion
```

Reply:
For sure, thanks!